src/cdk/stepper/stepper.ts

```diff
@@ -30,6 +30,12 @@ import {AbstractControl} from '@angular/forms';
 /** Used to generate unique ID for each stepper component. */
 let nextId = 0;
 
+/**
+ * Position state of the content of each step in horizontal stepper that is used for transitioning
+ * the content into correct position upon step selection change.
+ */
+export type CdkStepContentPositionState = 'left' | 'center' | 'right';
```

Comment:
Yes, it would need to once the `Directionality` related things are added to stepper, but I haven't added any code dealing with `Directionality`. I'll probably address them all at once, since it will also influence other interactions such as `_foucsIndex` management for horizontal stepper.